src/stores/AccountController.ts

```diff
@@ -69,10 +66,15 @@ export const handleVisibilityChange = () => {
 };
 
 export const initializeAccount = async () => {
-  await withAccountService(async ({ accountService, config }) => {
+  await useService(async ({ accountService, config }) => {
+    if (!accountService) {
```

Comment:
I don't understand this if case. Can you add a comment?

Reply:
sure, this is only for TVOD or configs without auth integration. But now, I can see that this function is called only for AUTHVOD and SVOD, so I believe that we can remove this if statement. The initial idea was to prevent requesting account services if client integration is not provided.